README.md

```diff
@@ -28,36 +28,59 @@ On top of displaying each link to you, the Link Diver will send http get request
  
 Link Diver will also filter links for you based on a regular expression you provide or by some of the metadata mentioned earlier.
  
-If you are only interested in filtering by a regular expression, simply write your expression into the textbox at the top of the page and press enter. Link Diver will then only display to you links that contain that regular expression somewhere in the link and will highlight the matches in red for each link. If however you want to filter in combination with other parameters, you will have to enter your regular expression in JSON format, like so:
-> {"regex":"^https://www[.]google[.]com"}.
+If you are only interested in filtering by a regular expression, simply write your expression into the textbox at the top of the page and press enter. Link Diver will then only display to you links that contain that regular expression somewhere in the link and will highlight the matches in red for each link. If however you want to filter in combination with other parameters, you will have to use the regexp modifier, like so:
+> regexp:^https://www[.]google[.]com
  
-To filter links by one of (or a combination of) their metadata attributes, you will also have to use a JSON format. Simply make the JSON key the metadata attribute you want to filter by, make the value the result you desire. For example if I wanted to filter for links coming from an anchor tag, I would type:
-> {"tagName":"A"}
+To filter links by one of (or a combination of) their metadata attributes, you can use a similar format. Each metadata attribute has it's own modifier which are listed below. Simply write the modifier of the metadata attribute you want to filter by, followed by a comma and the value you want. For example if I wanted to filter for links coming from an anchor tag, I would type:
+> tag:A
+
+To use a combination of these filtering arguments simply seperate the arguments with a space. For example to filter for links that are not visible and had a successful status code in combination with a regular expression you could do:
+> regexp:^https://www[.]google[.]com visible:false status_ok:true
  
-The following are all metadata you can filter by:
+The following are all of the modifieres you can use to filter:
  
+- regexp (Regular Expression)
 - host (Host)
 - visible (Visible)
-- tagName (Tag Source)
-- contentType (Content Type)
-- status (Status Code)
-- statusOk (Whether or not the status code is indicative of a succes or an error)
+- tag (Tag Source)
+- content_type (Content Type)
+- status_code (Status Code)
+- status_ok (Whether or not the status code is indicative of a succes or an error)
```

Comment:
no status_ok is suppose to mean whether or not status is a success or failure. If the status has not been fetched yet, than this is simply undefined, we could add some kind of has_status though

Reply:
I don't quite understand how it is different than status_code: 200, if it's redundant we might want to remove it

Reply:
Status code can be any number from 0-999, but in addition to the code the server also passes a boolean of whether the call was successful or not. Typically 200 is successful and 404 is an error. In short, status_code is the number between 0-999 and status_ok is a boolean indicating whether that number is a success or failure

But there are also other codes that can be successful or failures, so it's just a different way to categorize the status. the main use case of using status_ok instead of status_code is if you don't care about particular error codes and just want to separate out all errors generally.

But if you did actually want to distinguish between 401 and 404 (for example), you would want to use status_code instead of status_ok.